static/src/js/components/CollectionResults/CollectionResultsItem.js

```diff
@@ -201,176 +196,14 @@ export const CollectionResultsItem = forwardRef(({
-                                    />
-                                    Temporal subset
-                                  </li>
-                                )
-                              }
-                              {
-                                hasVariables && (
-                                  <li>
-                                    <EDSCIcon
-                                      className="collection-results-item__tooltip-feature-icon"
-                                      title="A white tags icon"
-                                      size="0.725rem"
-                                      icon={FaTags}
-                                    />
-                                    Variable subset
-                                  </li>
-                                )
-                              }
-                              {
-                                hasTransforms && (
-                                  <li>
-                                    <EDSCIcon
-                                      className="collection-results-item__tooltip-feature-icon"
-                                      title="A white horizontal sliders icon"
-                                      size="0.725rem"
-                                      icon={FaSlidersH}
-                                    />
-                                    Transform
-                                  </li>
-                                )
-                              }
-                              {
-                                hasFormats && (
-                                  <li>
-                                    <EDSCIcon
-                                      className="collection-results-item__tooltip-feature-icon"
-                                      title="A white file icon"
-                                      size="0.725rem"
-                                      icon={FaFileAlt}
-                                    />
-                                    Reformat
-                                  </li>
-                                )
-                              }
-                              {
-                                hasCombine && (
-                                  <li>
-                                    <EDSCIcon
-                                      className="collection-results-item__tooltip-feature-icon"
-                                      title="A white boxes icon"
-                                      size="0.725rem"
-                                      icon={FaCubes}
-                                    />
-                                    Combine
-                                  </li>
-                                )
-                              }
-                            </ul>
-                          </>
-                        )
-                      }
-                    />
+                  getCustomizeIcons(
```

Comment:
You wanna rename this getCustomizedIcons or getCustomIcons?

Reply:
This is for getting the icons that show how a download is customizable (e.g. concatenation, spatial subsetting, temporal, etc.)
I'm not sure `getCustomizedIcons` or `getCustomIcons` is more clear than `getCustomizeIcons`. Maybe `getCustomizableIcons`? But that could be confusing because just looking at the name someone might think the icons are customizable, but it's the icons indicating how the data is customizable.

Reply:
Whichever you think is right is fine. @eudoroolivares2016 if you could chime in I'd appreciate another brain on this,